build-system/server/app-utils.js

```diff
@@ -127,51 +127,51 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
     }
   } else if (mode == 'compiled') {
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.js/g,
-      hostName + '/dist/v0.js'
+      /https:\/\/cdn\.ampproject\.org\/v0\.(m?js)/g,
+      hostName + '/dist/v0.$1'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.js/g,
-      hostName + '/dist/shadow-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.(m?js)/g,
+      hostName + '/dist/shadow-v0.$1'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.js/g,
-      hostName + '/dist/amp4ads-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.(m?js)/g,
+      hostName + '/dist/amp4ads-v0.$1'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.js/g,
-      hostName + '/dist/video-iframe-integration-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.(m?js)/g,
+      hostName + '/dist/video-iframe-integration-v0.$1'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).js/g,
-      hostName + '/dist/v0/$1.js'
+      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).(m?js)/g,
```

Comment:
Nit: Wouldn't this regex be more correct if it were `\.`? `.` by itself means something completely different, although I guess it's not wrong.

Reply:
yeah i think the \ should be there, will fix